Rocket.Chat/Extensions/UIColorExtension.swift

```diff
@@ -19,6 +19,24 @@ extension UIColor {
         )
     }
 
+    // MARK: Color from strings (good|warning|danger)
+
+    static func normalizeColorFromString(string: String) -> UIColor {
+        if string == "warning" {
```

Comment:
What do you think of using a switch here instead?

Reply:
I think that using a switch would be better too. Also, what do you guys think about having a `NamedColor` enum holding the color names as member values and the colors itself as properties?